src/net.h

```diff
@@ -524,6 +560,22 @@ class CNode
         return false;
     }
 
+    // BUIPXXX:
+    bool GrapheneCapable()
+    {
+        if (nServices & NODE_GRAPHENE)
+            return true;
+        return false;
+    }
+
+    // BUIP055:
+    bool BitcoinCashCapable()
```

Comment:
this method is not present anymore in the `dev` branch

Reply:
Removed.